exercises/diamond/example.rs

```diff
@@ -0,0 +1,68 @@
+static ABC: &'static str = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+use std::ascii::AsciiExt;
+
+pub fn get_diamond(diamond_char: char) -> Vec<String> {
+    let diamond_char = AsciiExt::to_ascii_uppercase(&diamond_char);
+    if ABC.find(diamond_char).is_none() {
+        panic!("{:?} is not a valid char.", diamond_char);
```

Comment:
I get the motivation here, but it's best to leave panics as a worst case after absolutely everything else has failed. It makes more sense to me to just return an empty vector in this case.